src/libraries/Microsoft.Extensions.Configuration.Abstractions/src/Microsoft.Extensions.Configuration.Abstractions.csproj

```diff
@@ -1,8 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

Comment:
why are these changes needed?

Reply:
`netstandard2.0` has very limited nullable support (if any, I don't remember). NetCore target framework is needed for proper nullable annotation (some nullable attributes were just added in net5, for example).

- Here's the comment asking to add `$(NetCoreAppCurrent)` along with nullable annotations: https://github.com/dotnet/runtime/issues/57200#issuecomment-897710642
- And here's the issue for that: https://github.com/dotnet/runtime/issues/54012 (also mentioned in PRs description)